source/scss/_pf-basic-icons.scss

```diff
@@ -112,6 +112,7 @@
   content: $pficon-var-import;
 }
 .#{$pf-icon-prefix}-info:before {
+  color: $color-pf-black-700;
```

Comment:
The 4 alert icons have a different color than the normal gray. 
The declaration for this color was in the alerts.less. I moved together with the others in the icons file.